SpatialGDK/Extras/schema/authority_intent.schema

```diff
@@ -7,3 +7,10 @@ component AuthorityIntent {
      // 0 is reserved as an invalid/unset value.
      uint32 virtual_worker_id = 1;
 }
+
+component_set AuthorityIntentSet {
+  id = 9980;
+  components = [
+    AuthorityIntent
+  ];
+}
```

Comment:
Nit: indentation (4 spaces everywhere else). Also applies to other schema changes, ultimately doesn't matter that much though.

Reply:
Should probably add a linter for our handwritten schema.

Reply:
Follow up - https://improbableio.atlassian.net/browse/UNR-4489